src/ui/components/Center.tsx

```diff
@@ -17,6 +18,10 @@ export const Center = withState((props: CenterProps) => {
       onClick={props.playOrPause}
       onDoubleClick={props.toggleFullscreen}
     >
+      {props.logo.url && (
+        <div className={'igui_logo ' + ((props.logo.position && props.logo.position[1] == 'right') ? 'igui_logo_right' : '') + ' ' + ((props.logo.position && props.logo.position[0] == 'bottom') ? 'igui_logo_bottom' : '') } style={{ 'backgroundImage': 'url("' + props.logo.url + '")' }}>
```

Comment:
Can you use the `classnames` lib for readability? Also have a style variable defined above. This line gets pretty long.